packages/eds-core-react/src/components/Autocomplete/Autocomplete.test.tsx

```diff
@@ -175,6 +175,7 @@ describe('Autocomplete', () => {
     const onChange = jest.fn()
     render(
       <StyledAutocomplete
+        optionLabel={(label: string) => label}
```

Comment:
Do we need to add this to the tests?

Reply:
after updating styled-components yes, that's what I am saying above. It is probably a bug in sc